production/kubernetes/agent-loki-windows.yaml

```diff
@@ -0,0 +1,362 @@
+apiVersion: v1
```

Comment:
The yaml manifests are generated from Tanka configs so we'll need to set those up to generate this. Could you summarize what you've done here/anything else you've needed to add to get this to work?

Was this file given to any users or do you have it here as an example?

Reply:
This is meant to be used in a kubernetes cluster that has windows nodes. Nothing special needs to be done apart from `kubectl apply -f agent-loki-windows.yaml`.

The statefulset will be deployed only on windows node, so if you also have linux nodes you need an other statefulset for that.